openfisca_us/variables/contrib/taxsim/v11.py

```diff
@@ -9,4 +9,4 @@ class v11(Variable):
     definition_period = YEAR
 
     def formula(tax_unit, period, parameters):
-        return tax_unit("taxable_ui", period)
+        return tax_unit("tax_unit_taxable_unemployment_insurance", period)
```

Comment:
If we're renaming it, should we go with `unemployment compensation`, the term in [IRS documents](https://www.irs.gov/forms-pubs/exclusion-of-up-to-10200-of-unemployment-compensation-for-tax-year-2020-only) and the US Code?